docs/multicluster/architecture.md

```diff
@@ -103,3 +103,23 @@ Multi-cluster admins can also specify certain ClusterNetworkPolicies to be repli
 ClusterSet. The ACNP to be replicated should be created as a ResourceExport in the leader cluster, and
 the resource export/import pipeline will ensure member clusters receive this ACNP spec to be replicated.
 Each member cluster's Multi-cluster Controller will then create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Data plane
```

Comment:
Why call it data plane, but not "multi-cluster connectivity" or "cross-cluster connectivity", or even "multi-cluster tunnel"?

Reply:
I renamed it as `Antrea Multi-cluster Service Connectivity`, hope it's ok.